internal/node/node_repositories.bzl

```diff
@@ -285,7 +296,9 @@ def _yarn_download_impl(repository_ctx):
     urls to use.
 
     package_json: a list of labels, which indicate the package.json files that the npm label will point to.
-  
+
+    preserve_symlinks: Turn on --node_options=--preserve-symlinks for nodejs_binary and nodejs_test rules
```

Comment:
maybe share the doc using a constant?

Reply:
and why is there so much duplication here?

Reply:
There are the 4 variants now. npm_download_runtime, npm_local_runtime, yarn_download, yarn_local, which are all very similar.

Reply:
Can't think of how to reduce the duplicity. Linter will probably complain if the doc is not inline